Add content from community doc PR:1052 (#190)
rancher/longhorn-product-docs: docs/version-1.7.0/modules/en/pages/troubleshooting-maintenance/troubleshooting.adoc

----
touch /var/log/longhorn_driver.log
----

== Common issues

=== Volume can be attached/detached from UI, but Kubernetes Pod/StatefulSet etc cannot use it

==== Using with Flexvolume Plugin

> Check if the volume plugin directory has been set correctly. This is automatically detected unless user explicitly set it.

Author: Added the suggestions.
Thanks.

By default, Kubernetes uses `/usr/libexec/kubernetes/kubelet-plugins/volume/exec/`, as stated in the https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md/#prerequisites[official document].

Some vendors choose to change the directory for various reasons. For example, GKE uses `/home/kubernetes/flexvolume` instead.

> The correct directory can be found by running `ps aux|grep kubelet` on the host and check the `--volume-plugin-dir` parameter. If there is none, the default `/usr/libexec/kubernetes/kubelet-plugins/volume/exec/` will be used.

Reviewer: ```suggestion
You can find the correct directory by running `ps aux|grep kubelet` on the host and checking the `--volume-plugin-dir` parameter. If the parameter is empty, Longhorn uses the default directory `/usr/libexec/kubernetes/kubelet-plugins/volume/exec/`.
```

Reviewer: Use active voice sentences whenever possible.

Author: Noted
Thanks
